.travis.yml

```diff
@@ -1,5 +1,8 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.4.2
-before_install: gem install bundler -v 1.16.1
+  - 2.4.5
+  - 2.5.3
+before_install:
+  - gem update --system
+  - gem install bundler -v 1.17.1
```

Comment:
Why do we need to specify the version of bundler?

Reply:
We don't. Addressed in 0e090f3.